resources/check.rb

```diff
@@ -43,7 +43,7 @@
 property :runtime_assets, Array
 property :stdin, [true, false], default: false
 property :subdue, Hash
-property :subscriptions, Array, default: [], required: true
+property :subscriptions, Array, default: []
```

Comment:
since there are no standalone checks so I think we should always require someone to specify a subscription for a check.

Reply:
This one is also because of cookstyle: https://www.rubydoc.info/gems/cookstyle/RuboCop/Cop/Chef/ChefRedundantCode/PropertyWithRequiredAndDefault

Like the check command, this was just done to be the least disruptive. Given the documentation, this definitely should be required: (Array of Sensu entity subscriptions that check requests will be sent to. The array cannot be empty and its items must each be a string.)